terraform/deployments/govuk-publishing-platform/rails_assets.tf

```diff
@@ -0,0 +1,13 @@
+locals {
+  workspace_transformation = terraform.workspace == "default" ? "ecs" : terraform.workspace
```

Comment:
I wonder if this would be nicer, since this `workspace_transformation` local is only used twice and in both cases it's the same interpolation?

```terraform
bucket_name = join("-", [
  "govuk",
  var.govuk_environment,
  terraform.workspace == "default" ? "ecs" : terraform.workspace,
  "rails-assets"
])
```

Reply:
I think this code will be pushed up somewhere else as this is used in many places in the code base.

Reply:
True, but we already have `workspace_suffix`, `workspace_external_domain` etc. for that. Adding another form of the same thing doesn't improve the situation, unless you want to actually pull this up as part of this PR (which I think would be scope creep).

Reply:
I will pull it up in another PR, I need to fix some issues with the origin later anyway.